torchvision/models/optical_flow/raft.py

```diff
@@ -517,6 +517,19 @@ def forward(self, image1, image2, num_flow_updates: int = 12):
 
 
 class Raft_Large_Weights(WeightsEnum):
+    """The metrics reported here are as follows.
+
+    ``epe`` is the "end-point-error" and indicates how far (in pixels) the
+    predicted flow is from its true value. This is averaged over all pixels
+    of all images. ``per_image_epe`` is similar, but the average is different:
+    the epe is first computed on each image independently, and then averaged
+    over all images. This corresponds to "Fl-epe" (sometimes written "F1-epe")
+    in the original paper, and it's only used on Kitti. ``fl-all`` is also a
+    Kitti-specific metric, defined by the author of the dataset and used for the
+    Kitti leaderboard. It corresponds to the average of pixels whose epe is
+    either <3px, or <5% of flow's 2-norm.
```

Comment:
I wanted this to be a `..note ::` but it looks like it doesn't render properly. It's likely because we treat these docstrings are treated differently from the rest.